packages/components/src/components/spinner/spinner.stories.ts

```diff
@@ -1,75 +1,69 @@
+ */
 
 export default {
   title: 'Components/sd-spinner',
   component: 'sd-spinner',
+  tags: ['!dev'],
   args,
   argTypes,
   parameters: { ...parameters },
   decorators: [withActions] as any
 };
 
-/**
- * Default: This shows sd-spinner in its default state.
- */
-
 export const Default = {
-  render: (args: any) => {
-    return generateTemplate({ args });
-  }
+  name: 'Default',
+  render: () => html`<sd-spinner></sd-spinner>`
 };
 
 /**
- * Use the `color` attribute to change the color of the spinner and correspond (currentColor) with the parents color.
+ * Offers `color` variants for alternate experiences:
+ * - `currentColor` (default): used to inherit the color from the parent element
+ * - `primary`: used on light backgrounds
+ * - `white`: used on primary backgrounds
  */
 
-export const Color = {
+export const Variants = {
+  name: 'Variants',
   parameters: { controls: { exclude: 'color' } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: { type: 'attribute', name: 'color' }
-      },
-      options: {
-        templateBackgrounds: {
-          alternate: 'x',
-          colors: ['', 'rgb(var(--sd-color-primary, 0 53 142))', '#F6F6F6']
-        }
-      },
-      args
-    });
-  }
+  render: () =>
+    html` <div class="flex flex-row items-center gap-12">
+      <div class="bg-neutral-100 p-4">
+        <sd-spinner color="currentColor"></sd-spinner>
+      </div>
+      <div class="p-4">
+        <sd-spinner color="primary"></sd-spinner>
+      </div>
+      <div class="bg-primary p-4">
+        <sd-spinner color="white"></sd-spinner>
+      </div>
+    </div>`
 };
 
 /**
- * Use the font-size in css to scale the spinner.
+ * Use the `font-size` in css to scale the spinner proportionally.
  */
 
-export const Sizing = {
-  parameters: { controls: { exclude: ['color'] } },
-  render: (args: any) => {
-    return html`
-      ${generateTemplate({
-        axis: {
-          x: {
-            type: 'template',
-            name: 'individual sizing',
-            values: [
-              { value: '<div style="font-size: inherit">%TEMPLATE%</div>', title: 'font-size: inherit' },
-              { value: '<div style="font-size: 1rem">%TEMPLATE%</div>', title: 'font-size: 1rem' },
-              { value: '<div style="font-size: 2rem">%TEMPLATE%</div>', title: 'font-size: 2rem' },
-              { value: '<div style="font-size: 4rem">%TEMPLATE%</div>', title: 'font-size: 4rem' }
-            ]
-          }
-        },
-        args
-      })}
-    `;
-  }
+export const Size = {
+  name: 'Size',
+  render: () =>
+    html`<div class="flex flex-row gap-12">
+      <div style="font-size: inherit">
+        <sd-spinner color="currentColor"></sd-spinner>
+      </div>
+      <div style="font-size: 2rem">
+        <sd-spinner color="currentColor"></sd-spinner>
+      </div>
+      <div style="font-size: 4rem">
+        <sd-spinner color="currentColor"></sd-spinner>
+      </div>
```

Comment:
In Figma the size story has color="primary"
<img width="707" alt="image" src="https://github.com/user-attachments/assets/18fd2ab0-99a3-47ad-8a0b-32ebf07b9555">

Reply:
The primary is the default color in figma, but not the default color in code. Currently, I used the default color but I can make it primary if it makes more sense

Reply:
Updated